src/utils/components/ModalWithBackdrop.vue

```diff
@@ -119,11 +120,48 @@ export default {
             type: Boolean,
             default: false,
         },
+        movable: {
+            type: Boolean,
+            default: false,
+        },
     },
     emits: ['close'],
     data() {
         return {
             hideForPrint: false,
+            isDragging: false,
+            dragStartX: 0,
+            dragStartY: 0,
+            modalLeft: 0,
+            modalTop: 0,
+            modalClass: '',
+        }
+    },
+    computed: {
+        headerStyle() {
+            return {
+                cursor: this.movable ? 'move' : 'default',
+            }
+        },
+        modalStyle() {
+            const transform = this.top ? '' : 'translate(0%, -50%)'
+            return {
+                top: this.modalTop + 'px',
+                left: this.modalLeft + 'px',
+                transform: transform,
+            }
+        },
+    },
+    mounted() {
+        this.modalClass = {
+            'start-50': true,
+            'top-50 translate-middle': !this.top,
+            'translate-middle-x on-top-with-padding': this.top,
+        }
+        if (this.movable) {
+            this.$nextTick(() => {
+                this.setInitialPosition()
+            })
         }
```

Comment:
In the `mapPopopver` module, we handle the movable aspect of the component with a `useMovableElement` function, like so : https://github.com/geoadmin/web-mapviewer/blob/693e590984ed0dc3c71db15179bc2f2434c92c52/src/modules/map/components/MapPopover.vue#L83-L113

I think this would be good to keep only one way to move popups in the viewer.